packages/concerto-core/index.js

```diff
@@ -102,6 +102,9 @@ const Concerto = require('./lib/concerto');
 // MetaModel
 const MetaModel = require('./lib/introspect/metamodel');
 
+// ConcertoCodes
+const ConcertoCodes = require('./lib/concertoCodes');
```

Comment:
We already have error codes, I believe. Should we consolidate with this? Or rename this to be deprecation-specific?

Reply:
Let's consolidate. I see that you're modifying Error Code already in this PR.

The Warning Codes should be part of concerto-util

Reply:
Yes moved them there